examples/tutorial/jupyter/execution/pandas_on_dask/local/exercise_2.ipynb

```diff
@@ -50,11 +50,10 @@
    "source": [
     "### Dataset: 2015 NYC taxi trip data\n",
     "\n",
-    "Link to raw dataset: https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page\n",
     "\n",
     "We will be using a version of this data already in S3, originally posted in this blog post: https://matthewrocklin.com/blog/work/2017/01/12/dask-dataframes\n",
     "\n",
-    "**Size: ~2GB**"
```

Comment:
We can still use the `dask-data` dataset here because it is replaced with the smaller `modin-test` datatset in our CI (same as in the `pandas_on_ray` case)